exporter_app/tasks/export_ead_task.rb

```diff
@@ -46,11 +58,36 @@ def load_into_work_queue(updates)
     #
     # -- Mark (Wednesday 20 April  15:38:14 AEST 2016)
     #
+    # Still don't seem to need it ...
+    #
+    # -- James (Fri Apr 29 14:31:38 AEST 2016)
+    #
     updates['removes'].each do |remove_id|
       @work_queue.push('remove', remove_id)
     end
 
     @work_queue.optimize
   end
 
+  def ead_export_directory
+    exp_dir = File.join(@workspace_directory, EXPORTED_DIR)
+    Dir.mkdir(exp_dir) unless Dir.exist?(exp_dir)
+    exp_dir
+  end
+
+  def ead_export_file(id)
+    File.join(ead_export_directory, "#{id}.xml")
+  end
+
+  def download_ead(id, repo_id, feed)
+    open File.join(ead_export_file(id)), 'w' do |io|
```

Comment:
Better to use `File.open` instead of `open` here, and some extra parentheses on the `File.open` and `io.write` would help my brain if you have a spare couple of pairs.